build.gradle

```diff
@@ -204,9 +205,11 @@ task getRevision() {
 			remote = 'origin/main'
 		}
 		def revision = grgit.log(includes:[remote]).size()
-		file('build/revision.txt').text = revision.toString().trim()
+		def revString = revision.toString().trim()
+		file('build/revision.txt').text = revString
 		// Update the version to the new revision
-		project.setVersion(revision.toString().trim())
+		project.setVersion(revString)
+		println("KoLmafia version :" + project.getVersion())
```

Comment:
This prints as:

    KoLmafia version :12345-M

Is that intended?

Reply:
Hmm.  Probably could be another PR, but I generally want to be able to run "./gradlew getRevision and check my revision.  Right now you can call the function and get nothing.  This is also a carryover from ANT functionality.  Will put it back in my sandbox.

Reply:
Oh, my comment was more that it prints "version :12345-M" and not "version: 12345-M"

Reply:
ah.  will fix.